clients/openshift.go

```diff
@@ -23,9 +23,11 @@ import (
 	authoapi "github.com/openshift/api/authorization/v1"
 	"github.com/openshift/api/image/v1"
 	networkoapi "github.com/openshift/api/network/v1"
+	routeoapi "github.com/openshift/api/route/v1"
 	authv1 "github.com/openshift/client-go/authorization/clientset/versioned/typed/authorization/v1"
 	imagev1 "github.com/openshift/client-go/image/clientset/versioned/typed/image/v1"
 	networkv1 "github.com/openshift/client-go/network/clientset/versioned/typed/network/v1"
+	routev1 "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1"
```

Comment:
So the two imports are intentional? Just make sure since I can see how goimports would confuse the two.

Reply:
Yeah they are intentional. Was following the approach for the `network`, `image`, and `auth` resources.